### PR TITLE
PPT-9 Created compensation for byes case in bracket creation.

### PR DESCRIPTION
**Pull request by cpboone**

### CODE REVIEW

**Comment by cbolgiano** [comment]
Looks good to me!!

File: test/pingpong/bracket_test.clj [hunk 1]
```diff
@@ -0,0 +1,15 @@
+(ns pingpong.bracket_test
+    (:require [clojure.test :refer :all]
+            [pingpong.bracket :refer :all]))
+
+(deftest byes-test
+  (testing "Failed generating bracket containing byes."
+    (is 
+      (= #pingpong.bracket.Bracket{:left #pingpong.bracket.Bracket{:left #pingpong.bracket.Bracket{:left nil, :right nil, :game {:p1 "bob", :p2 nil}}, :right #pingpong.bracket.Bracket{:left nil, :right nil, :game {:p1 "bill", :p2 "jack"}}, :game {:p2 nil, :p1 nil}}, :right #pingpong.bracket.Bracket{:left #pingpong.bracket.Bracket{:left nil, :right nil, :game {:p1 "monkey", :p2 nil}}, :right #pingpong.bracket.Bracket{:left nil, :right nil, :game {:p1 "jill", :p2 "james"}}, :game {:p2 nil, :p1 nil}}, :game {:p2 nil, :p1 nil}}
```
Comment (AnthonyRinaldi) [inline test/pingpong/bracket_test.clj]: Nice!

**Comment by AnthonyRinaldi** [comment]
Looks good to me!

**Comment by kettermanm** [comment]
Good job!

**Pull request merged**